rules/do-you-use-loggermessage-in-net.md/rule.md

```diff
@@ -0,0 +1,118 @@
+---
+seoDescription: Learn how to optimize logging in .NET applications using the LoggerMessage class. Discover the performance benefits, such as reduced object allocations and efficient message parsing, and get practical tips on how to implement this pattern for better logging efficiency and structured logs in high-performance scenarios.
+type: rule
+title: Do you use LoggerMessage in .NET?
+uri: do-you-use-loggermessage-in-net
```

Comment:
Hi @tomek-i 

- [ ] 1. Can you make the URL shorter? E.g. Ditch the "do-you-"

Let me know if you need help with this (we can call on Teams). There are instructions in the Wiki, but basically you need to update the URL and the folder. If it was an existing rule, then a redirect would be necessary, but that is not the case.